packages/frontend-api/src/Resources/config/graphql-types/ModelType/Transport/TransportDecorator.types.yaml

```diff
@@ -38,8 +38,16 @@ TransportDecorator:
             payments:
                 type: "[Payment!]!"
                 description: "List of assigned payments"
+                resolve: '@=query("paymentsOfTransportQuery", value, args["displayInCartOnly"])'
+                args:
+                    displayInCartOnly:
+                        type: "Boolean"
+                        defaultValue: true
```

Comment:
for the sake of unification, shouldn't the Payment::transports also accept the argument displayInCartOnly?